cclib/parser/psi4parser.py

```diff
@@ -1197,6 +1221,11 @@ def parse_vibration(n, inputfile):
         line = next(inputfile)
         assert 'RMS dev' in line
 
+        line = next(inputfile)
+        assert 'IR activ' in line
+        chomp = line.split()
+        vibirs = [utils.float(x) for x in chomp[3:]]

```

Comment:
Even though older versions won't be able to parse this, in order to have the return value of the method be consistent, you can probably return the empty list when they aren't available.